lms/static/coffee/src/instructor_dashboard/data_download.coffee

```diff
@@ -87,6 +90,39 @@ class DataDownload
           @$reports_request_response.text data['status']
           $(".msg-confirm").css({"display":"block"})
 
+    @$list_issued_certificate_table_btn.click (e) =>
+      url = @$list_issued_certificate_table_btn.data 'endpoint'
```

Comment:
@mattdrayer  @andy-armstrong Let me know, For the change related to SOL-1288. I used the pre-existing file "data_download.coffee" that include the data downloading section stuff under instructor dashboard. 

Let me know is it acceptable for you to write / update this file with coffee scripts ?